src/NLog.Targets.Redis/NLog.Targets.Redis.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45;netstandard1.5</TargetFrameworks>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```

Comment:
This requires a change to the below conditionals.

Reply:
Conditionals have been removed.